e4tools/bundles/org.eclipse.e4.tools.emf.editor3x/src/org/eclipse/e4/tools/emf/editor3x/UndoAction.java

```diff
@@ -75,20 +75,23 @@ private void update() {
 	 */
 	public static String getCommandLabel(Command cmd) {
 
-		if (cmd instanceof SetCommand) {
-			SetCommand sc = (SetCommand) cmd;
+		switch (cmd) {
+		case SetCommand sc -> {
 			return sc.getLabel() + " " + sc.getFeature().getName() + " on " + sc.getOwner().eClass().getName(); //$NON-NLS-1$ //$NON-NLS-2$
-		} else if (cmd instanceof AddCommand) {
-			AddCommand ac = (AddCommand) cmd;
+		}
+		case AddCommand ac -> {
 			return ac.getLabel() + " " + getFirstClassName(ac.getCollection()); //$NON-NLS-1$
-		} else if (cmd instanceof DeleteCommand) {
-			DeleteCommand dc = (DeleteCommand) cmd;
+		}
+		case DeleteCommand dc -> {
 			Collection<?> deleted = dc.getCollection();
 			if (deleted.size() == 1) {
 				return dc.getLabel() + " " + getFirstClassName(deleted); //$NON-NLS-1$
 			}
 			return dc.getLabel() + " " + dc.getCollection().size() + " Objects"; //$NON-NLS-1$ //$NON-NLS-2$
 		}
+		case null, default -> {
```

Comment:
I'm not sure I understand why null and default are specified, Isn't default sufficient.  Is something generating these changes?